packages/bpk-component-breadcrumb/src/BpkBreadcrumb.js

```diff
@@ -26,24 +26,71 @@ import STYLES from './BpkBreadcrumb.scss';
 
 const getClassName = cssModules(STYLES);
 
+type SchemaMetaDataItem = {
+  url: string,
+  label: string,
+};
+
 export type Props = {
   children: Node,
+  schemaMetaData: ?(SchemaMetaDataItem[]),
   label: string,
 };
 
+/*
+  The google structured data reference for the stringified output of
+  the following function is here:
+  https://developers.google.com/search/docs/data-types/breadcrumb
+*/
+const buildMetaData = (schemaMetaData: SchemaMetaDataItem[]): string => {
+  const itemListElement = schemaMetaData.map((schemaMetaDataItem, index) => ({
+    '@type': 'ListItem',
+    position: index + 1,
+    item: {
+      '@id': schemaMetaDataItem.url,
+      name: schemaMetaDataItem.label,
+    },
+  }));
+
+  return JSON.stringify({
+    '@context': 'http://schema.org',
+    '@type': 'BreadcrumbList',
+    itemListElement,
+  });
+};
+
 const BpkBreadcrumb = (props: Props) => {
-  const { children, label, ...rest } = props;
+  const { children, label, schemaMetaData, ...rest } = props;
 
   return (
-    <nav aria-label={label} {...rest}>
-      <ol className={getClassName('bpk-breadcrumb')}>{children}</ol>
-    </nav>
+    <React.Fragment>
+      {Array.isArray(schemaMetaData) && (
+        <script
+          type="application/ld+json"
+          // eslint-disable-next-line react/no-danger
+          dangerouslySetInnerHTML={{ __html: buildMetaData(schemaMetaData) }}
```

Comment:
@mrkalndz  one question here: How often does the schema metadata change? Can we avoid re calculating on every render? Just from an efficiency perspective.

If it doesn't change then maybe we can store this so we don't have to re compute every time